man/crypttab.xml

```diff
@@ -250,6 +250,15 @@
         option.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>sector-size=</option></term>
+
+        <listitem><para>Specifies the sector size in bits. See
```

Comment:
Could you make this a bit more clear. For example:

`Specifies the sector size in bytes for use with disk encryption in plain mode.`

Reply:
We should not really copy the cryptsetup manpage here, as the cryptsetup (8) reference has the details about the option, including caveats etc.